### DISCUSSION TITLE
How to delete msg + 1?

### DISCUSSION THREAD

**Discussion by EMPTYyYyYyY** [nextcord/nextcord#928] [Q&A] [answered]
Hi, guys! **Thanks in advance more for your work**, I decided to rewrite my bot after the Discord API stopped working.
Question: How can I delete using the command I created the command message or how many previous messages I want + text when I wrote the command in the chat.

**Еxample(as it was before):**
```
@commands.command()
    async def clear(self, ctx, amount: int):  
        messages = await ctx.channel.purge(limit=amount + 1)
```
**Еxample(now):**
```
async def clear(ctx):
     msg = await ctx.send("удалить это сообщение")
     await msg.delete()
```
but it only deletes the message I sent, and I just want the command to delete the messages. please tell me how you can implement my old example. 
if you show the code with the implementation, I will be the happiest person in the world.

**Comment by DWAA1660** [comment, marked as answer]
The old code will work exactly the same

**Reply by EMPTYyYyYyY** [reply]
It isnt work(

**Reply by EMPTYyYyYyY** [reply]
I'm idiot... It works
```
@bot.command(pass_context=True)
async def clear(ctx, amount: int):
    await ctx.channel.purge(limit=amount+1)
```
